addons/web/static/src/views/form/form_controller.scss

```diff
@@ -305,6 +304,18 @@
 
     // Statusbar
     .o_form_statusbar {
+        position: relative;
+        top: unset;
+        z-index: 0;
+
+        @include media-breakpoint-up(md) {
+            position: sticky;
+            top: 0;
+            z-index: 1;
+            background-color: $body-bg;
+            box-shadow: 0 0.3rem 0.25rem -0.25rem rgba(0, 0, 0, 0.075);
```

Comment:
Would be nice to use an existing value... `$box-shadow-sm` ?

Reply:
Unfortunately bootstrap box-shadows also appear on the sides of the element which looks pretty ugly
![image](https://github.com/user-attachments/assets/1ea753df-6605-4336-aa74-3e17dc902911)

The configuration I've used allows to avoid this.

Reply:
Makes sense. :+1: